Gemfile.lock

```diff
@@ -105,19 +105,19 @@ GEM
     marcel (0.3.3)
       mimemagic (~> 0.3.2)
     method_source (0.9.2)
-    mimemagic (0.3.3)
+    mimemagic (0.3.4)
     mini_mime (1.0.2)
     mini_portile2 (2.4.0)
-    minitest (5.13.0)
-    msgpack (1.3.1)
+    minitest (5.14.0)
+    msgpack (1.3.3)
     nio4r (2.5.2)
-    nokogiri (1.10.7)
+    nokogiri (1.10.8)
       mini_portile2 (~> 2.4.0)
-    popper_js (1.14.5)
-    public_suffix (4.0.2)
+    popper_js (1.16.0)
+    public_suffix (4.0.3)
```

Comment:
### [public_suffix](https://simonecarletti.com/code/publicsuffix-ruby)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 4.0.2 → 4.0.3

[_~~change-log~~, [source-code](http://github.com/weppos/publicsuffix-ruby)_]





<details>
<summary>Commits</summary>

A change of **7** commits. See the full changes on [the compare page](https://github.com/weppos/publicsuffix-ruby/compare/4.0.2...4.0.3).

These are the individual commits:
- (af3244f) [Update rubocop](https://github.com/weppos/publicsuffix-ruby/commit/af3244f9cf1c558a4c6afd3e39a8ac46c7495c77)
- (ec6bc1d) [Experiment with https://keepachangelog.com/](https://github.com/weppos/publicsuffix-ruby/commit/ec6bc1dc040cd1517fa2e5012eca658a3c908eb6)
- (d076009) [Update .travis.yml](https://github.com/weppos/publicsuffix-ruby/commit/d076009df0160292dab73eadf13021f5da55761d)
- (bac3a1d) [Fix 2.7 deprecations and warnings (#168)](https://github.com/weppos/publicsuffix-ruby/commit/bac3a1d05c26e5fc1a8dd7919fad4d8a4ca39163)
- (41f54ea) [CHANGELOG for GH-167](https://github.com/weppos/publicsuffix-ruby/commit/41f54eab90e2c7088ed1cdce2eebb900e0197f15)
- (da650c8) [Welcome 2020](https://github.com/weppos/publicsuffix-ruby/commit/da650c860a552336113d73d77b25b724feb8e9c8)
- (84fc0d1) [Release 4.0.3](https://github.com/weppos/publicsuffix-ruby/commit/84fc0d1895178aea481d00529697322f87a82fba)


</details>